recipes/clove-unit/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

```suggestion
conan_basic_setup(TARGETS)

find_package(CloveUnit CONFIG REQUIRED)

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} CloveUnit::CloveUnit)
```

CONAN_LIBS will be deprecated on Conan 2.0